static/src/javascripts/projects/common/modules/identity/email-preferences.js

```diff
@@ -0,0 +1,122 @@
+// This is a workaround for the email preferences page https://profile.thegulocal.com/email-prefs
+// We want to submit subscribe/unsubscribe requests without a full page refresh
+// Hopefully this will be short-lived; if it is still alive in 2017, git blame and cry
+
+define([
+    'bean',
+    'qwery',
+    'reqwest',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    qwery,
+    reqwest,
+    fastdom,
+    $
+) {
+    function useReqwest(buttonEl) {
```

Comment:
Not sure about the name of this function. Might be clearer to have a `bindClickHandlers` function and separate `postFormData` function?